DfE.FindInformationAcademiesTrusts/DfE.FindInformationAcademiesTrusts.csproj

```diff
@@ -11,6 +11,7 @@
     <ItemGroup>
         <PackageReference Include="Azure.Extensions.AspNetCore.DataProtection.Keys" Version="1.2.4" />
         <PackageReference Include="Azure.Identity" Version="1.12.0" />
+        <PackageReference Include="ClosedXML" Version="0.102.3" />
```

Comment:
Would be good to have an ADR about why we're using this library, especially given that the authors currently describe it as "not stable"

Reply:
Primarily due to it's licensing (MIT) and it's prevalence (75m downloads and 15.5k everyday).  EPPlus the other popular alternative incurs cost and the other most popular library NPOI has some political issues.

Due to it's prevalence and open source nature the risk is minimal as it's effectively a wrapper around OpenXML.

It would be good to get a link to where they state it's not stable, considering how widely it's used! Was this a specific release?

Reply:
To that end I'll look into, given how simple our use case is, how much complexity/time would be involved in just using OpenXML.

Reply:
I'm comfortable using this library after our conversation this morning. I'll write an ADR to that affect after the fact (within the next couple days).

Reply:
Added task to create ADR (so it doesn't get forgotten)